plat/kvm/arm/entry64.S

```diff
@@ -36,11 +36,13 @@
 #include <uk/config.h>
 #include <uk/reloc.h>
 
+#define BOOTSTACK_SIZE	4096
```

Comment:
In commit message of `plat/kvm: Do not unconditionally clean and invalidate the cache`:
s/[1[/[1]/
s/inalidate/invalidate